src/labels_transfer/knn/test.py

```diff
@@ -0,0 +1,151 @@
+import re
+import subprocess
+import pytest
+from pathlib import Path
+import anndata as ad
+import mudata as mu
+import numpy as np
+from scipy.sparse import csr_matrix
+
+## VIASH START
+meta = {
+    'resources_dir': './resources_test/'
+}
+## VIASH END
+
+reference_h5ad_file = f"{meta['resources_dir']}/annotation_test_data/TS_Blood_filtered.h5ad"
+# convert reference to h5mu
+reference_adata = ad.read_h5ad(reference_h5ad_file)
+reference_mdata = mu.MuData({"rna": reference_adata})
+reference_file = f"{meta['resources_dir']}/annotation_test_data/TS_Blood_filtered.h5mu"
+reference_mdata.write_h5mu(reference_file)
+input_file = f"{meta['resources_dir']}/pbmc_1k_protein_v3/pbmc_1k_protein_v3_filtered_feature_bc_matrix.h5mu"
+
+
+def test_label_transfer(run_component):
```

Comment:
Could you use the `random_path` fixture to create the oputput path? Example here:
https://github.com/openpipelines-bio/openpipeline/blob/4c373aabd035e7af9169716a8ffa73dfd240816c/src/mapping/cellranger_multi/test.py#L67C42-L67C53